docs/operator/multi_vo_rucio.md

```diff
@@ -4,21 +4,21 @@ title: Multi-VO Rucio
 sidebar_label: Multi-VO Rucio
 ---
 
-This section provides an overview of using Rucio for multiple virtual
-organizations (VOs) on a single instance with the "Multi-VO" feature. Rather
-than each VO having to set up an entire instance by themselves, they can share
-the same server and database which are run by a central "super_root", and
-continue to use Rucio as they would a normal or "Single-VO" instance. Their
-accounts, scopes and RSEs are associated with their VO which ensures all rules
-and replicas are kept separate from other VOs using the instance.
```

Comment:
I am personally not sure if this section needed to be changed - the previous version seems clear enough to me